packages/testcontainers/__tests__/containers/RegTestContainer/index.test.ts

```diff
@@ -37,5 +37,10 @@ describe('regtest', () => {
       const address = await container.getNewAddress('not-default', 'legacy')
       expect(address.length).toStrictEqual(34)
     })
+
+    it('should be able to getnewaddress with label and as eth', async () => {
+      const address = await container.getNewAddress('not-default', 'eth')
+      expect(address.length).toStrictEqual(42)
```

Comment:
hmm.. personally prefer to have ethersjs support to validate the addr here
```
ethers.utils.isAddress(address);
```
or using `regex` to validate is good as well